tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -233,25 +243,59 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     return {"granules": granules_list, "copied_to_glacier": copied_file_urls}
 
 
-def get_default_glacier_bucket_name(config: Dict[str, Any]) -> str:
+def get_destination_bucket_name(config: Dict[str, Any]) -> str:
+    try:
```

Comment:
Add a quick doc string on what the function does.

Reply:
Added